sql/hive-thriftserver/v2.3/src/main/java/org/apache/hive/service/cli/operation/OperationManager.java

```diff
@@ -86,20 +86,15 @@ private void initOperationLogCapture(String loggingMode) {
   }
 
   public ExecuteStatementOperation newExecuteStatementOperation(HiveSession parentSession,
```

Comment:
Why do you need to replace the previous methods here?

Reply:
I addressed you comment in #29933, so could you check it here?

Reply:
I think this update is just to pass `queryTimeout` into `newExecuteStatementOperation`.